.idea/code42cli.iml

```diff
@@ -10,10 +10,10 @@
     <option name="requirementsPath" value="" />
   </component>
   <component name="PyDocumentationSettings">
+    <option name="format" value="GOOGLE" />
```

Comment:
would you mind .gitignoring this file and removing it from the repo? It unfortunately doesn't at all work the way I hoped and we ended up removing it from py42.

Reply:
sure